cranelift/codegen/src/isa/aarch64/inst/args.rs

```diff
@@ -613,6 +613,14 @@ pub enum VectorSize {
 }
 
 impl VectorSize {
+    /// Get the vector operand of the same size but with 8-bit lane size.
+    pub fn as_scalar8_vector(&self) -> VectorSize {
+        match self.is_128bits() {
```

Comment:
For a small-enough enum like this, to avoid any surprises in the future (e.g., add 256-bit vectors and mistake "not 128 bits" as meaning 64 bits rather than 256), I'd prefer a full match over all cases if possible; or alternately, a match on number of bits, and a separate `bits` method that has a full match.

Reply:
good catch :)